stock_prices/stock_prices.py

```diff
@@ -2,14 +2,30 @@
 
 import argparse
 
+
 def find_max_profit(prices):
-  pass
+    min_price_so_far = float("inf")
+    current_max_profit = float("-inf")
+    max_profit = float("-inf")
+    for p in prices:
+        current_max_profit = p - min_price_so_far
```

Comment:
Nice! Hopefully figuring this one out got you into the problem-solving flow and will make the next on a bit easier.